src/pages/Filing/FilingApp/FieldEntry.tsx

```diff
@@ -104,6 +101,10 @@ function FieldEntry({
     setCurrentPage(inputNumber);
   };
 
+  // Selectively enable full table borders based on if the table's wrapper div is overflowing
+  const [tableDivReference, tableDivReferenceIsOverflowing] =
+    useIsOverflowing();
```

Comment:
Does this run on every table, even ones we know won't be wider than the screen? Like for single field?